### DISCUSSION TITLE
Ao criar conteúdo pt-BR sobre tecnologias en-US, traduzir e contextualizar termos ou manter original?

### DISCUSSION THREAD

**Discussion by estevanmaito** [frontendbr/forum#1671] [Discussão]
Minha questão difere dos tópicos já levantados aqui sobre escrever em pt ou en.

> Se tratando de um conteúdo exclusivamente pt-BR, você prefere/acha mais agradável um texto 100% pt-BR, que tenta traduzir os termos técnicos usados ou que mantém algumas coisas na língua nativa (geralmente en-US)?

**Um exemplo:**

1) Uma forma de atualizar o estado no React é usando o método `setState`. // Vou criar um componente de função.

2) Uma forma de atualizar o state no React é usando o método `setState`. // Vou criar um function component.

**Minha motivação**: eu estou criando conteúdo e tenho usado sempre que possível a primeira forma, ainda que não tenha problemas em compreender nenhuma das duas, mas esse sou eu, que entendo inglês.

*se não for incômodo, quem puder informar seu próprio nível de inglês, vai ajudar a ter uma noção.

**Comment by portothree** [comment]
Percebo que vai muito de como a tecnologia foi introduzida e é disseminada na comunidade. No seu exemplo eu preferiria mesmo "estado", tanto por ser o mais comum em artigos de react no brasil quanto por já deixar claro que está a modificar o estado da aplicação. 

Costumo traduzir algumas documentações e artigos e também acontece de ter essa dúvida, mas por ler conteúdos das duas comunidades, consigo identificar quais são expressões em comum como, por exemplo, "binding".

Percebo que a ideia de ter exclusivamente em português é para produzir um texto mais inclusivo, mas a finalidade da linguagem é a comunicação ser efetiva, se a convenção da comunidade é manter alguma palavra em inglês e traduz, provavelmente vai causar mais confusão, já que é provável que o seu artigo não será o único do assunto a ser visto pela pessoa. Trabalhar uma boa analogia parece-me mais válido.

**Comment by luizeboli** [comment]
Acho que vai um pouco do que o @portothree comentou, os termos vêm de como a tecnologia foi disseminada.

Usando de exemplo o próprio React, pra mim é bem estranho ler a documentação traduzida e encontrar "Fragmentos" (referência ao Fragment) ou "Hook de efeito" (useEffect).

Não é tanto questão de saber inglês ou não, e sim entender o que determinado termo significa. Como uma convenção...

**Comment by ninetails** [comment]
Eu acho que vai muito de bom senso. Tem coisas que a gente fala estrangeirizado e já está enraizado na comunidade; como tem outras que já exista uma tradução que não deturpe a definição.

**Comment by tarciozemel** [comment]
Traduzo artigos técnicos em `en` há muitos anos e, para casos como o que você exemplificou, coloco a expressão original seguida de sua tradução entre parênteses (ou vice-versa) _na primeira vez em que ela aparece no texto_.

A partir daí, é possível usar intercambiando sem problemas de não-entendimento, já que ambos já foram supracitados (o que também dá uma mãozinha com SEO).
